massa-client/src/repl.rs

```diff
@@ -50,29 +50,32 @@ pub(crate) async fn run(client: &Client, wallet: &mut Wallet) {
             let parameters = input[1..].to_vec();
             // Print result of evaluated command
             match cmd {
-                Ok(command) => println!("{}", command.run(client, wallet, &parameters, true).await),
-                Err(_) => println!("{}", Command::not_found()),
+                Ok(command) => match command.run(client, wallet, &parameters, false).await {
+                    Ok(output) => output.pretty_print(),
+                    Err(e) => println!("{}", style(format!("Error: {}", e)).red()),
```

Comment:
In every case? or only if the error is of the `WrongParameters` kind? because here we could fail for a lot of reasons, and many are more related to the `node` than misusage of the user!

Reply:
Only for the WrongParameter kind